CHANGELOG.md

```diff
@@ -347,9 +347,6 @@
     - Clips not in scale mode are unaffected (similar to the existing transpose behaviour from the encoders).
     - Configureable in `SETTINGS > MIDI > TRANSPOSE` between chromatic and in-scale transposition.
 
-#### <ins>Loopback</ins>
-- Added a `MIDI LOOPBACK` mode, accessible in the SONG menu, which directs MIDI data from internal MIDI clips back to the Deluge input.
-
```

Comment:
instead of just deleting the entry, we should add an entry that says it was removed.

Reply:
> instead of just deleting the entry, we should add an entry that says it was removed.

Since we never officially released this feature I don't think we need to, unless it's a note for folks that use in the nightly and beta builds.

For 1.1 we removed the feature. 

It got left in the 1.1 notes in the nightly by mistake and should've been moved up to the 1.2 section.